drivers/sensor/lsm6dsr/lsm6dsr.c

```diff
@@ -0,0 +1,754 @@
+/* lsm6dsr.c - Driver for LSM6DSR accelerometer, gyroscope and
+ * temperature sensor
+ */
+
+/*
+ * Copyright (c) 2023, Andrei Hutanu
```

Comment:
Shouldn't the copyright of the code this was copied from be retained?

This is nearly identical to the code in lsm6dsl.c.

Reply:
Hi, I've changed the copyright to reflect mostly lsm6dso.c work. Please confirm that you agree w/ this. Regards.